vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/model/SequenceOverview.java

```diff
@@ -0,0 +1,40 @@
+package edu.asu.diging.vspace.core.services.impl.model;
+
+import java.util.List;
+
+import edu.asu.diging.vspace.core.model.ISlide;
+
+public class SequenceOverview {
+ 
+    private String name;
+
+    private String id;
+    
+    private List<SlideOverview> slideOverview;
+
+    public List<SlideOverview> getSlides() {
+        return slideOverview;
+    }
+
+    public void setSlides(List<SlideOverview> slideOverview) {
+        this.slideOverview = slideOverview;
+    }
```

Comment:
getters and setters need to be properly named (`getSlideOverview` if the variable is named `slideOverview`)